game/screens/credits.py

```
@@ -229,21 +281,40 @@ def _build_sections(self):
     def _measure_content(self, max_row_width):
         y = 0
         y += self._title_height(font_size=34, pad_x=10, pad_y=8) + 24
-        for role, entries in self._build_sections():
+        # Per-section, one-name-per-line measurement
+        name_font = 34
+        role_font = 18
+        pad_x = 10
+        pad_y = 8
+        spacing = 16
+        role_header_font = 36
+        for section, entries in self._build_sections():
```

Ruff (0.15.12)

[warning] Loop control variable `section` not used within loop body

Rename unused `section` to `_section`

(B007)